tests/terraform/runner/test_plan_runner.py

```diff
@@ -68,6 +68,26 @@ def test_runner_two_checks_only(self):
         self.assertEqual(report.get_summary()["failed"], 3)
         self.assertEqual(report.get_summary()["passed"], 3)
 
+    def test_tf_plan_filtered_rule(self):
```

Comment:
can you add a matching test for `terraform` as well (not just `tf_plan`)

Reply:
We have pretty thorough TF tests for this case here: `tests/terraform/checks/resource/aws/test_S3AbortIncompleteUploads.py`
That's why I added the tf_plan part, which is what was missing in our coverage and would've caught this.